_episodes/12-virtual-environments.md

```diff
@@ -63,7 +63,7 @@ which contains a particular version of Python installation
 plus a number of additional external libraries.
 
 Virtual environments are not just a feature of Python -
```

Comment:
```suggestion
Virtual environments are not just a feature of Python---
```

Reply:
I like this proposal - think it explains things more clearly - thank you @bielsnohr.